lib/src/value/calculation.dart

```diff
@@ -604,6 +604,21 @@ final class SassCalculation extends Value {
     }
   }
 
+  /// Creates an `calc-size()` calculation with the given [basis] and [value].
+  ///
+  /// The [basis] and [value] must be either a [SassNumber], a
```

Comment:
Value is optional right? So it can also be null

Reply:
Yes, but that's implicit from the type.